doc/source/en/TortoiseGit/tgit_dug/dug_commit.xml

```diff
@@ -113,17 +113,14 @@
 			</para>
 		</important>
 	</sect2>
-	<!--
 	<sect2 id="tgit-dug-commit-changelists">
 		<title>Change Lists</title>
 		<para>
-			The commit dialog supports Git's changelist feature to help
+			The commit dialog supports changelist feature to help
 			with grouping related files together. Find out about this feature
 			in <xref linkend="tgit-dug-changelists"/>.
 		</para>
 	</sect2>
-	-->
-	<!--
 	<sect2 id="tgit-dug-commit-ignore">

```

Comment:
Would be great if we could also mention the Git flags `assume-valid` and `skip-worktree`.